SU-SOE/stanford_news
# D8CORE-1915: Theming the vertical cards

# READY FOR REVIEW

# Summary
- Styling the vertical cards on the news node and the ones added to paragraphs.

# Review Needed By (Date)
- May 1

# Urgency
- Critical.

# Steps to Test

1. Pull in this branch.
2. Clear caches.
3. Turn off the URL routing on articles so it doesn't route you off the news page.
4. Duplicate a news article so there is a total of 4.
5. Look at the vertical cards at the bottom of an article. You should have three minimal cards across.
6. Add a News View to paragraph. Select News. Then select Vertical Teaser Term. In the advanced options, add 1 -3 items. 
7. Repeat so you have examples of 1, 2, and 3.  Verify they look right on desktop and mobile.

<img width="1644" alt="Screen Shot 2020-04-29 at 11 52 25 AM" src="https://user-images.githubusercontent.com/1187335/80654397-fca5a980-8a30-11ea-8b88-2fcd614c6e47.png">


# Affected Projects or Products
- SOE, Stanford Basic
- News Accelerant

# Associated Issues and/or People
- D8CORE-1915
- D8CORE-1918
- @boznik 
- @sherakama 

# Known things.
- Please review this ticket for more details: https://stanfordits.atlassian.net/browse/D8CORE-1915
- There are a couple of questions and changes that should be reviewed.
